src/library/librarycontrol.h

```diff
@@ -147,6 +153,7 @@ class LibraryControl : public QObject {
     // Library widgets
     WLibrary* m_pLibraryWidget;
     WLibrarySidebar* m_pSidebarWidget;
+    WSearchLineEdit* m_pSearchbox;
```

Comment:
Segmentation fault on startup because this plain pointer is not initialized with `nullptr` in the constructor!

Reply:
Thank you! Must have slipped throught when I split up the lib-control-fix PR.